classes/help.php

```diff
@@ -146,7 +145,7 @@ function print_page() {
 								<a name="general_options"></a>
 								<div class="postbox">
 
-									<h3><?php _e( 'General Options', 'SimpleMap' ); // XSS ok. ?></h3>
+									<h3><?php _e( 'General Options', 'simplemap' ); // XSS ok. ?></h3>
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->